mlshvch/hometask-5/1.sql

```diff
@@ -0,0 +1,3 @@
+SELECT CONCAT(first_name, ' ', last_name) AS actors
```

Comment:
a) The alias is not needed if you are not referencing it anywhere;
b) It's not clearly stated in the task, but I think `SELECT * ...` should be enough. Otherwise you've lost the `actor_id` and `last_updated` columns. But once again, the exact list of returned columns is not specified in the task, so it's a minor thing.